code/game/gamemodes/vampire/vampire_bite.dm

```diff
@@ -0,0 +1,22 @@
+/mob/living/carbon/human/proc/handle_vamp_biting(var/mob/living/carbon/human/M)
+	var/datum/antagonist/vampire/V = is_vampire(M)
+	if(!V || M == src || M.zone_selected != "head")
+		return FALSE
+	if((NOBLOOD in dna.species.species_traits) || dna.species.exotic_blood || !blood_volume)
+		to_chat(M, "<span class='warning'>They have no blood!</span>")
+		return FALSE
+	if(is_vampire(src))
+		to_chat(M, "<span class='warning'>Your fangs fail to pierce [name]'s cold flesh</span>")
+		return FALSE
+	if(is_species(src, /datum/species/skeleton))
```

Comment:
would it be better to check for no blood?
```suggestion
	if(is_species(src, NOBLOOD))
```
or something? Other wise you have to add in like plasma men

Reply:
i'll look into it but i'm not sure that's how the noblood check works.